<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimParamDialog</class>
 <widget class="QDialog" name="SimParamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>462</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>382</width>
    <height>462</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>382</width>
    <height>462</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Set Parameters</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="optLabel">
        <property name="text">
         <string>Optimization Scheme</string>
        </property>
        <property name="buddy">
         <cstring>optComboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="optComboBox">
        <item>
         <property name="text">
          <string>Utility</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Expected Utility</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fallbacks</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Extended Fallbacks</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="utilLabel">
        <property name="text">
         <string>Utility Scheme</string>
        </property>
        <property name="buddy">
         <cstring>utilComboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="utilComboBox">
        <item>
         <property name="text">
          <string>Transplants</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 Year Survival</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 Year Survival</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Score</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="chainLengthLabel">
        <property name="text">
         <string>Maximum Chain Length</string>
        </property>
        <property name="buddy">
         <cstring>chainLengthSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="chainLengthSpinBox">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="chainLengthSlider">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Numerical Parameters</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>341</width>
          <height>223</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="pairFailureRateLabel">
             <property name="text">
              <string>Pair Failure Rate</string>
             </property>
             <property name="buddy">
              <cstring>chainLengthSpinBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="pairFailureRateSpinBox">
             <property name="decimals">
              <number>2</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="adFailureRateLabel">
             <property name="text">
              <string>AD Failure Rate</string>
             </property>
             <property name="buddy">
              <cstring>chainLengthSpinBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="adFailureRateSpinBox">
             <property name="decimals">
              <number>2</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="exogenousFailureRateLabel">
             <property name="text">
              <string>Exogenous Match Failure Rate</string>
             </property>
             <property name="buddy">
              <cstring>chainLengthSpinBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="exogenousFailureRateSpinBox">
             <property name="decimals">
              <number>2</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="praBox">
           <property name="text">
            <string>Add Advantage to High PRA Recipients</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QSpinBox" name="praCutoffSpinBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="praCutoffLabel">
             <property name="text">
              <string>Cutoff</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="praAdvantageSpinBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="praAdvantageLabel">
             <property name="text">
              <string>Value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QSpinBox" name="solutionsSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>25</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="solutionsLabel">
             <property name="text">
              <string>Number of Solutions</string>
             </property>
             <property name="buddy">
              <cstring>solutionsSpinBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="solutionsSlider">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>25</number>
             </property>
             <property name="pageStep">
              <number>2</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Additional Options</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>11</y>
          <width>314</width>
          <height>189</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Chains Are Stored:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="chainStorageComboBox">
             <item>
              <property name="text">
               <string>As They Are Found</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>First</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Last</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="reserveOtoOBox">
           <property name="text">
            <string>Reserve O Donors for O Recipients</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkDPBox">
           <property name="text">
            <string>Check Additional HLA Information</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="compatibleBox">
           <property name="text">
            <string>Include Compatible Pairs in Match Run</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="excludeABDonorsBox">
           <property name="text">
            <string>Exclude AB Donors from Simulation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="allowABBridgeBox">
           <property name="text">
            <string>Allow AB Bridge Donors</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>chainLengthSpinBox</tabstop>
  <tabstop>chainLengthSlider</tabstop>
  <tabstop>utilComboBox</tabstop>
  <tabstop>optComboBox</tabstop>
  <tabstop>praBox</tabstop>
  <tabstop>reserveOtoOBox</tabstop>
  <tabstop>checkDPBox</tabstop>
  <tabstop>compatibleBox</tabstop>
  <tabstop>allowABBridgeBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SimParamDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SimParamDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chainLengthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>chainLengthSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chainLengthSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>chainLengthSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>solutionsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>solutionsSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>solutionsSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>solutionsSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excludeABDonorsBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SimParamDialog</receiver>
   <slot>enableBridgeDonorOptions(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>praBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SimParamDialog</receiver>
   <slot>enablePRAOptions(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enablePRAOptions(bool)</slot>
  <slot>enableBridgeDonorOptions(bool)</slot>
 </slots>
</ui>
